ca/ca.go

```diff
@@ -415,7 +414,7 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 	serialBytes[0] = byte(ca.prefix)
 	_, err = rand.Read(serialBytes[1:])
 	if err != nil {
-		err = core.InternalServerError(err.Error())
+		err = berrors.InternalServerError("boulder/ca.IssueCertificate: failed to generate serial: %s", err)
```

Comment:
> A method name variable may make sense, although I'd probably use a function local var in order not to pollute the global namespace any more.
That makes more sense :+1: 

> So I just realized the reason I did this was to differentiate errors from the go-grpc/grpc package from ones from our boulder/grpc package. Looking at it now I think it probably doesn't actually matter since we include the method name on the end and the other package just uses the prefix grpc:.

Ahh, ok. That makes sense. I think its safe to remove but if anyone else feels differently I wouldn't make it a fight.